sdk/resourcemanager/azure-resourcemanager/src/test/java/com/azure/resourcemanager/TestVirtualMachineSsh.java

```diff
@@ -59,26 +54,6 @@ public VirtualMachine createResource(VirtualMachines virtualMachines) throws Exc
 
         Assertions.assertNotNull(vm.innerModel().osProfile().linuxConfiguration().ssh());
         Assertions.assertTrue(vm.innerModel().osProfile().linuxConfiguration().ssh().publicKeys().size() > 0);
-
-        JSch jsch = new JSch();
-        Session session = null;
-        if (TestUtils.isRecordMode()) {
-            try {
-                java.util.Properties config = new java.util.Properties();
-                config.put("StrictHostKeyChecking", "no");
-                // jsch.addIdentity(sshFile, filePassword);
-                session = jsch.getSession("testuser", publicIpDnsLabel + "." + "eastus.cloudapp.azure.com", 22);
-                session.setPassword(password);
```

Comment:
Delete this SSH part. No idea why create VM with SSH then use password to connect.